manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/tooling/Document.scala

```diff
@@ -202,6 +205,24 @@ case class Query(queryText: String, assertions: QueryAssertions, myInitQueries:
     content.runnableContent(initQueries ++ myInitQueries :+ queryText)
 }
 
+case class ConsoleData(globalInitQueries: Seq[String], localInitQueries: Seq[String], query: String) extends Content with NoQueries {
+  override def asciiDoc(level: Int): String = {
+    val globalInitQueryRows = globalInitQueries.mkString(NewLine)
+    val localInitQueryRows = localInitQueries.mkString(NewLine)
+    s""".Try this query live
+       |[console]
+       |----
+       |$globalInitQueryRows
+       |
```

Comment:
I'd think this blank line will mess things up.
We don't have it in the old code: https://github.com/neo4j/neo4j/blob/3.0/manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DocumentingTestBase.scala#L530

In fact, I'd prefer if we stay away from using the "console" asciidoc.py extension (we need to remove it) and instead generate the DocBook output it will create for us. Here's what we could include instead:
```
ifndef::nonhtmloutput[]
++++
<formalpara role="cypherconsole">
<title>Try this query live</title>
<para><database>
QUERIES TO INIT THE GRAPH GOES HERE
</database><command>
THE QUERY GOES HERE
</command></para></formalpara>
++++
endif::nonhtmloutput[]
```